coredns.yaml

```diff
@@ -25,17 +25,21 @@ environment:
 pipeline:
   - uses: git-checkout
     with:
-      expected-commit: 8868454177bdd3e70e71bd52d3c0e38bcf0d77fd
+      expected-commit: ae2bbc29be1aaae0b3ded5d188968a6c97bb3144
       repository: https://github.com/coredns/coredns
       tag: v${{package.version}}
 
   - runs: |
       # Ensures plugins get included
       make check
 
+  - uses: go/bump
+    with:
+      deps: golang.org/x/net@v0.17.0 google.golang.org/grpc@v1.58.3 golang.org/x/crypto@v0.17.0 github.com/quic-go/quic-go@v0.37.7
+
   - uses: go/build
     with:
-      go-package: go-1.20
+      go-package: 1.20
```

Comment:
```suggestion
```

Reply:
Hey If you are trying to use the --go-version=1.20 of go/bump. It might be a mistake but Its always looks for the platform go version no matter what we specify